servicetalk-dependencies/build.gradle

```diff
@@ -26,24 +26,23 @@ javaPlatform {
 dependencies {
   api platform(project(":servicetalk-bom"))
   api platform("io.netty:netty-bom:${nettyVersion}")
-  api platform("org.glassfish.jersey:jersey-bom:$jerseyVersion")
+  api platform("com.fasterxml.jackson:jackson-bom:${jacksonVersion}")
+  api platform("com.google.protobuf:protobuf-bom:${protobufVersion}")
+  api platform("org.apache.logging.log4j:log4j-bom:${log4jVersion}")
+  api platform("org.glassfish.jersey:jersey-bom:${jerseyVersion}")
 
   constraints {
     // Use `api` only for dependencies whose types appear in ServiceTalk API.
-    api "com.fasterxml.jackson.core:jackson-databind:$jacksonVersion"
     api "com.google.api.grpc:proto-google-common-protos:$protoGoogleCommonProtosVersion"
-    api "com.google.protobuf:protobuf-java:$protobufVersion"
     api "io.opentracing:opentracing-api:$openTracingVersion"
     api "io.zipkin.reporter2:zipkin-reporter:$zipkinReporterVersion"
     // `spi` and `core` types exposed in `log4j2-mdc-utils`
-    api "org.apache.logging.log4j:log4j-core:$log4jVersion"
```

Comment:
Can we also remove `runtime "org.apache.logging.log4j:log4j-slf4j-impl:$log4jVersion"` from line 50?

Reply:
No, we want that dependency explicitly as a runtime-only dependency as it should not be included on the compile classpath.